### PR TITLE
mnk50: Configurator Fix and Layout Macro Additions

### PR DESCRIPTION
**Pull request by noroadsleft**
## Description

* adds an object for K36 (7u spacebar) to `info.json/LAYOUT_all`.
* applies human-friendly formatting to `info.json`
* adds `LAYOUT_split_space` and `LAYOUT_7u_space` macros

cc @yiancar (keyboard maintainer)

## Types of Changes

- [x] Enhancement/optimization
- [x] Keyboard (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mnk50/mnk50.h [hunk 1]
```diff
@@ -30,3 +45,27 @@
...
+#define LAYOUT_7u_space( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, \
+    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B,      K1D, \
+    K20,      K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2D, \
+         K31, K32,                K36,                K3A, K3B            \
+) { \
+    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, XXX, K1D }, \
+    { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, XXX, K2D }, \
+    { XXX, K31, K32, K33, XXX, K35, XXX, K37, XXX, K39, K3A, K3B, XXX, XXX }  \
```
Comment (yiancar) [inline keyboards/mnk50/mnk50.h:70]: this should be `{ XXX, K31, K32, XXX, XXX, XXX, K36, XXX, XXX, XXX, K3A, K3B, XXX, XXX } \`

**Review by yiancar — CHANGES_REQUESTED** [review]

**Comment by yiancar** [comment]
Thank you for the beautification:)

File: keyboards/mnk50/mnk50.h (hunk 1 above)
Comment (noroadsleft) [reply keyboards/mnk50/mnk50.h]: Good catch!

File: keyboards/mnk50/info.json [hunk 2]
```diff
@@ -6,7 +6,167 @@
...
+                {"x":1.25, "y":3},
+                {"x":2.25, "y":3, "w":1.25},
+                {"x":3.5, "y":3},
+                {"x":4.5, "y":3, "w":2.25},
+                {"x":6.75, "y":3, "w":2.75},
+                {"x":9.5, "y":3},
+                {"x":10.5, "y":3, "w":1.25},
+                {"x":11.75, "y":3}
+            ]
+        }
```
Comment (fauxpark) [inline keyboards/mnk50/info.json:118]:
```suggestion
        },
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**